src/libraries/System.Text.RegularExpressions/tests/RegexExperiment.cs

```diff
@@ -129,12 +129,12 @@ private static long MeasureMatchTime(Regex re, string input, out Match match)
 
         private void TestRunSampleRegex()
         {
-            string rawregex = @"(?<Part0>(every\ week))\s*(?<Part1>(on)){0,1}\s*((?<Part2>(\b([Mm]ondays|[Tt]uesdays|[Ww]ednesdays|[Tt]hursdays|[Ff]ridays|[Ss]aturdays|[Ss]undays)\b))|(?<Part3>((\b([Mm]onday|[Tt]uesday|[Ww]ednesday|[Tt]hursday|[Ff]riday|[Ss]aturday|[Ss]unday)\b)|(\b([Mm]on|[Tt]ues|[Tt]ue|[Ww]ed|[Tt]hurs|[Tt]hur|[Tt]hu|[Ff]ri|[Ss]at|[Ss]un)\b))))";
+            string rawregex = @"^[a-z]+\-[a-z]+$";
             string input = File.ReadAllText(inputfile);
-            TestRunRegex("sample", rawregex, input);
+            TestRunRegex("sample", rawregex, input, true, true);
```

Comment:
style nit, feel fee to ignore: we usually name bool parameters at the caller site as it's self documenting. Occasionally we do this when passing a null for the same reason.
```suggestion
            TestRunRegex("sample", rawregex, input, viewDGML:true, dotStar:true);
```

Reply:
Ok, I'll follow that style more consistently going forward, I've sometimes done that.